requirements.txt

```diff
@@ -1,16 +1,19 @@
-futures==3.2.0
-BeautifulSoup==3.2.1
-pexpect==4.5.0
-psutil==5.6.6
+future==0.18.2
+futures==3.1.1
```

Comment:
As I understand the `futures` package is a backport of the `concurrent.futures` builtin library in Python 3. Why do we still need it after we convert the project to Python 3?

Reply:
Hi Colin,

Thanks for pointing it out. Yes we should remove it, as we are not providing backward compatibility for python2.7